package.json

```diff
@@ -18,7 +18,7 @@
 		"url": "https://github.com/Automattic/woocommerce-subscriptions-core/issues"
 	},
 	"scripts": {
-		"postinstall": "composer install",
+		"postinstall": "is-ci || composer install",
```

Comment:
I'm not entirely clear why this change is needed. Is there harm `composer install` is run on `npm install`?

Reply:
No real harm but it's not necessary as we don't use composer dependencies in this workflow. I made this change mostly for performance reasons.